webapp-django/crashstats/crashstats/models.py

```diff
@@ -401,7 +398,7 @@ def fetch(
             self.cache_seconds
         ):
             name = implementation.__class__.__name__
-            cache_key = hashlib.md5(name + text_type(params)).hexdigest()
+            cache_key = hashlib.md5(name + six.text_type(params)).hexdigest()
```

Comment:
This won't work in Python 3 because `md5` works on `bytes` but this has been in production for a while and not causing issues, so it's probably the case it's "ok for now". I'll fix this when I get the webapp tests working in Python 3.